components/net/cookie_storage.rs

```diff
@@ -83,7 +83,9 @@ impl CookieStorage {
             Ok(None)
         }
     }
-
+    pub fn clear_storage(&mut self) {
+        self.cookies_map.clear();
```

Comment:
According to https://w3c.github.io/webdriver/#dfn-delete-cookies we should set the cookie expiry time to the past, rather than deleting the actual stored cookie.

Reply:
Okay I'll try to implement that :)

Reply:
@jdm a little help , what should I assign the value of cookie's expiry_time variable to?

Reply:
Try `now().to_timespec() - Duration::seconds(1)`?

Reply:
I dont think we need it in timespec since the type of expiry_time is Tm.